src/libs/feature-previews-config.ts

```diff
@@ -84,6 +85,17 @@ const featurePreviewsConfig: readonly FeaturePreview[] = [
     )}`,
     lastUpdated: '7/10/2025',
   },
+  {
+    id: WORKFLOW_RETRY_WITH_MORE_MEMORY,
+    title: 'Retry Workflow Tasks with More Memory',
+    description: 'Opt-in to enable access to the Retry with More Memory workflow submission setting.',
+    feedbackUrl: `mailto:dsp-analysis@broadinstitute.org?subject=${encodeURIComponent(
+      'Retry with More Memory Feedback'
+    )}`,
+    documentationUrl:
+      'https://support.terra.bio/hc/en-us/articles/39412460844699-Automatically-retrying-workflows-with-more-memory',
+    lastUpdated: '8/8/2025',
```

Comment:
Chosen as an estimated release date.

Reply:
I think it would be great to mention Java/JVM/`OutOfMemoryError` for those who don't click through to the article.

Reply:
What do you think of the wording now?

Reply:
Looks great to me! Thanks.